cmds/replay.c

```diff
@@ -1183,10 +1183,11 @@ int command_replay(int argc, char *argv[], struct opts *opts)
 
 	if (!opts->flat && peek_rstack(&handle, &task) == 0)
 		print_header(&output_fields, "#", "FUNCTION", 1, false);
-	if (opts->srcline)
-		pr_gray(" [SOURCE]");
-	pr_out("\n");
-
+	if (!list_empty(&output_fields)) {
+		if (opts->srcline)
+			pr_gray(" [SOURCE]");
+		pr_out("\n");
+	}
```

Comment:
Can you please keep the blank line before the while loop below?  It'd separate the header line from data.

Reply:
sorry. it's updated.